internal/mode/static/state/graph/reference_grant_test.go

```diff
@@ -347,10 +399,24 @@ func TestRefAllowedFrom(t *testing.T) {
 			toResource:     toService(notAllowedNsName),
 			expAllowed:     false,
 		},
+		{
+			name:           "ref allowed from tlsroute to service",
+			refAllowedFrom: refAllowedFromTLSRoute,
+			toResource:     toService(allowedTLSRouteNsName),
+			expAllowed:     true,
+		},
+		{
+			name:           "ref not allowed from tlsroute to service",
+			refAllowedFrom: refAllowedFromTLSRoute,
+			toResource:     toService(notAllowedNsName),
+			expAllowed:     false,
+		},
 	}
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
```

Comment:
```suggestion
		test := test
		t.Run(test.name, func(t *testing.T) {
```

you need to redefine the test variable inside the loop

Reply:
huh, how come?

Reply:
nvm I found some documentation